client/wfnews-war/src/main/angular/src/assets/data/appConfig.json

```diff
@@ -96,5 +96,15 @@
         "rest": {
             "url": "https://d1apps.vividsolutions.com/pub/wfim/wfim-api/v1"
         }
+    },
```

Comment:
Contact information will also need to be added to the config.jsp (thats where it comes from when deployed)